bodhi/tests/server/consumers/test_masher.py

```diff
@@ -1201,7 +1201,7 @@ def test_absent_gating(self, publish, *args):
                 cwd=t.mash_dir, shell=False, stderr=-1,
                 stdin=mock.ANY,
                 stdout=mock.ANY)])
-        self.assertEquals(len(t.state['completed_repos']), 1)
+        self.assertIsNotNone(t.state['completed_repo'])
```

Comment:
Can we change these ```self.assertIsNotNone```'s to ```self.assertEqual```'s? That way we have stronger assertions and can make sure that they are the right value too.